driver/src/main/java/org/cf/oracle/Driver.java

```diff
@@ -19,7 +19,7 @@
 
 public class Driver {
 
-    private static final String DRIVER_DIR = "/data/local";
+    private static final String DRIVER_DIR = "/data/local/tmp";
```

Comment:
On my phone, it's not the correct temp dir. :D

A better solution would be to determine the best directory.